src/main/java/app/komunumo/data/service/LoginService.java

```diff
@@ -122,7 +123,9 @@ private boolean internalLogin(final @NotNull String emailAddress) {
         }
 
         LOGGER.info("User with email {} successfully logged in.", emailAddress);
-        authenticationSignal.setAuthenticated(true, UserRole.ADMIN.equals(user.role()));
+        authenticationSignal.setAuthenticated(true, UserRole.ADMIN.equals(user.role()),
+                UserRole.USER.equals(user.role()) && UserType.LOCAL.equals(user.type()));
```

Comment:
The check if a user is a user can be omitted. A local user is always a user. If the associated role is `ADMIN`, it includes `USER`. So the check if the user type is local should be enough.

Reply:
Omitted the check.